build/repo.targets

```diff
@@ -26,6 +26,8 @@
   </PropertyGroup>
 
   <Target Name="PrepareOutputPaths">
+    <MakeDir Directories="$(ArtifactsDir);$(BuildDir);$(LineupBuildDir)" />
```

Comment:
Can these be condensed to one call to MakeDir instead of 3?

Reply:
There's a missing semicolon, that's what broke the build